intake_esm/config.yaml

```diff
@@ -117,9 +117,9 @@ collections:
       - ocean
       - ocnbgchem
       - seaIce
+default_chunk_size: 12MiB
```

Comment:
I don't know if this is a good value. Should we stick with the `dask` default as our default?

Reply:
Changed it to dask default of `128MiB`